### PR TITLE
added card component and added it to favorites page, for display purp…

### PR DESCRIPTION
**Pull request by majraoruc**
…oses

# Description

Please include a summary of the change for the appropriate task.

Task # (task link)

## Screenshots

Please add any relevant screenshots/screencaptures

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/Components/Card.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import Avatar from '@mui/material/Avatar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import {red} from '@mui/material/colors';
+import FavoriteIcon from '@mui/icons-material/Favorite';
+import ShareIcon from '@mui/icons-material/Share';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import MoreVertIcon from '@mui/icons-material/MoreVert';
+
+const ExpandMore = styled((props) => {
```
Comment (zlayabekrija) [inline src/Components/Card.js:18]: Let's try to keep one component declaration per file. Please move this component to appropriate place. Eg helper_components/Expandmore.js

File: src/Components/Card.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  return <IconButton {...other} />;
+})(({theme, expand}) => ({
+  transform: !expand ? 'rotate(0deg)' : 'rotate(180deg)',
+  marginLeft: 'auto',
+  transition: theme.transitions.create('transform', {
+    duration: theme.transitions.duration.shortest,
+  }),
+}));
+
+export default function RecipeReviewCard() {
```
Comment (zlayabekrija) [inline src/Components/Card.js:29]: Could we name this component as Movie card?

File: src/Components/Card.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+          <IconButton aria-label="settings">
+            <MoreVertIcon />
+          </IconButton>
+        }
+        title="Shrimp and Chorizo Paella"
+        subheader="September 14, 2016"
+      />
+      <CardMedia
+        component="img"
+        height="194"
```
Comment (zlayabekrija) [inline src/Components/Card.js:54]: How did we get this number? If we need the this, let's pass it as a props and maybe have it as default value 194.

File: src/Components/Card.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        <ExpandMore expand={expanded} onClick={handleExpandClick} aria-expanded={expanded} aria-label="show more">
+          <ExpandMoreIcon />
+        </ExpandMore>
+      </CardActions>
+      <Collapse in={expanded} timeout="auto" unmountOnExit>
+        <CardContent>
+          <Typography paragraph>Method:</Typography>
+          <Typography paragraph>Heat 1/2 cup of the broth in a pot until simmering, add saffron and set aside for 10 minutes.</Typography>
+          <Typography paragraph>
+            Heat oil in a (14- to 16-inch) paella pan or a large, deep skillet over medium-high heat. Add chicken, shrimp and chorizo, and
```
Comment (zlayabekrija) [inline src/Components/Card.js:80]: Let's pass all dynamic content as a prop and avoid hard coding.

File: src/Components/Card.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+        alt="Paella dish"
+      />
+      <CardContent>
+        <Typography variant="body2" color="text.secondary">
+          This impressive paella is a perfect party dish and a fun meal to cook together with your guests. Add 1 cup of frozen peas along
+          with the mussels, if you like.
+        </Typography>
+      </CardContent>
+      <CardActions disableSpacing>
+        <IconButton aria-label="add to favorites">
```
Comment (zlayabekrija) [inline src/Components/Card.js:65]: Please make this option toggleable. You can use state for the time being.

File: src/Components/Card.js [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+          <FavoriteIcon />
+        </IconButton>
+        <IconButton aria-label="share">
```
Comment (zlayabekrija) [inline src/Components/Card.js:68]: We might keep this, but lets give it some value.
We can utilize something like https://www.npmjs.com/package/react-share-social

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Thanks for contributing @majraoruc, code looks quite neet, but there are some small things to be covered, please address given comments. 
Thanks :) 
If you have any doubts please feel free to reach me out.

File: src/Components/Card.js (hunk 1 above)
Comment (majraoruc) [reply src/Components/Card.js:18]: i'll fix the folder structure per instructed

File: src/Components/Card.js (hunk 3 above)
Comment (majraoruc) [reply src/Components/Card.js:54]: keep in mind that I added the generic component just for visual's sake. We need to agree on a Movie Card design in order to make any viable changes

Comment (zlayabekrija) [reply src/Components/Card.js:54]: Agree, let's mark this as resolved

**Pull request closed without merging**